syntax = "proto3";

option cc_generic_services = true;

import "service/DataService.proto";
package apb.api.character;

option java_package = "jp.co.apb.server.api.proto.character";
option java_multiple_files = true;
option csharp_namespace = "Art.Framework.ApiNetwork.Grpc.Api.Character";

message RebirthRequest {
  int32 character_id = 1;
  int32 rebirth_count = 2;
}

message RebirthResponse {
  map<string, .apb.api.data.DiffData> diff_user_data = 99;
}

service CharacterService {
  rpc Rebirth(.apb.api.character.RebirthRequest) returns (.apb.api.character.RebirthResponse);
}

